app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -390,6 +392,19 @@ export default {
       }
     },
     message(updatedMessage) {
+      if (this.hasFiles === true) {
+        const el = this.$refs.advancedEditor.$el.querySelector(
+          '.ProseMirror-woot-style > p > img'
+        );
+        const imgElWrap = el.parentNode;
+        if (el) {
+          this.removeFocusEditorInputFileld();
+          this.$nextTick(() => {
+            imgElWrap.removeChild(el);
+          });
+        }
+        this.hasFiles = false;
+      }
```

Comment:
@iamsivin This seems like a hack, can disable paste event for files in the prosemirror editor